src/helpers.ts

```diff
@@ -192,7 +192,9 @@ const defaultLogLevel: LogLevel = (() => {
 export const logger = winston.createLogger({
   level: defaultLogLevel,
   exitOnError: false,
+  defaultMeta: { component: 'core-api' },
   format: winston.format.combine(
+    winston.format.metadata(),
```

Comment:
It adds a `metada` object to the log output where it can be used for API components filtering purpose. For instance:

`{"level":"info","message":"API server listening on: http://127.0.0.1:3999","metadata":{"component":"core-api"},"timestamp":"2023-04-17T16:49:45.955Z"}`
or
`{"level":"verbose","message":"new connection: rVVvoH_n-He7JvDIAAAB","metadata":{"component":"socket-io"},"timestamp":"2023-04-17T16:54:26.597Z"}`